.github/workflows/snap.yaml

```diff
@@ -2,7 +2,7 @@ name: Publish Snap
 
 # turn off edge snap builds temporarily and make it manual
 
-# on: 
+# on:
```

Comment:
Weird to have this change in this PR?

Reply:
Ah i ran prettier on the whole repo and it must have fixed that too. I'll remove and fix that seperately

Reply:
Yep, and we should do a refactor to change all the errors returned by the api to the new ApiError

Reply:
yes. Will add that to the project board